src/main/java/com/serenitydojo/calculator/Calculator.java

```diff
@@ -1,7 +1,68 @@
 package com.serenitydojo.calculator;
 
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+
 public class Calculator {
+
+    List<String> items = new ArrayList<>();
+
     public int evaluate(String expression) {
-        return 0;
+        int evaluvatedValue=0;
+
+        if (!(expression.isEmpty())){
+            items = Arrays.stream(expression.split(" "))
+                    .filter(val->!(val.isEmpty()))
+                    .collect(Collectors.toList());
+            evaluvatedValue = calculateValue(items);
+        }
+
+        return evaluvatedValue;
+
     }
+
+
+    private int calculateValue(List<String> strItems){
+
+        int itemCount = strItems.size();
+        int i=0 ;
```

Comment:
your formatting is all over the place. please go to "intelliJ -> file -> settings -> tools -> actions on save" and enable auto reformat 
![image](https://user-images.githubusercontent.com/2108984/232030041-cbb5ddc8-5fdd-47de-867a-d1740991c8bd.png)


then, when you make a change and save (ctrl+s), the file should look a bit cleaner :)

Reply:
My bad , you mentioned this before as well . I need to practice this for sure . Thanks